sdk/appcenter-distribute/src/test/java/com/microsoft/appcenter/distribute/DistributeDownloadTest.java

```diff
@@ -786,4 +786,17 @@ public Object answer(InvocationOnMock invocation) {
         /* Verify new release checked (for example what we installed was something else than the upgrade. */
         verify(mDialog, times(2)).show();
     }
+
+    @Test
+    public void completeWorkflowOnStartDownloading() {
```

Comment:
The unit test does not match the repro scenario.

Also should we just verify notifyUserConfirmation calls are correct instead?